net/wireguard-kmod/Makefile

```diff
@@ -1,5 +1,5 @@
 PORTNAME=	wireguard-kmod
-PORTVERSION=	0.0.20210502
+PORTVERSION=	0.0.20210503
```

Comment:
I already merged origin/main from upstream, please revert any change to this file

Reply:
Done